arduino-ide-extension/src/browser/dialogs/ide-updater/ide-updater-component.tsx

```diff
@@ -1,10 +1,15 @@
-import { nls } from '@theia/core/lib/common';
-import { shell } from '@theia/core/electron-shared/@electron/remote';
+import { nls } from '@theia/core/lib/common/nls';
 import * as React from '@theia/core/shared/react';
-import ReactMarkdown from 'react-markdown';
+// @ts-expect-error see https://github.com/microsoft/TypeScript/issues/49721#issuecomment-1319854183
+import type { Options } from 'react-markdown';
 import { ProgressInfo, UpdateInfo } from '../../../common/protocol/ide-updater';
 import ProgressBar from '../../components/ProgressBar';
 
+const ReactMarkdown = React.lazy<React.ComponentType<Options>>(
+  // @ts-expect-error see above
```

Comment:
I don't think this line is needed, as I'm getting this compilation error:

```

src/browser/dialogs/ide-updater/ide-updater-component.tsx:9:3 - error TS2578: Unused '@ts-expect-error' directive.

9   // @ts-expect-error see above
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

[12:50:39 PM] Found 1 error. Watching for file changes.
```

I've tried to remove it and the markdown in the dialog seems to be displayed correctly.

Reply:
> as I'm getting this compilation error:

in Code or with `tsc`? Without it, it does not compile for me in Code.

Reply:
I just pulled the latest changes and ran `yarn` again and it worked 👍